Application/Common/Interfaces/ISessionService.cs

```diff
@@ -11,4 +11,10 @@ public interface ISessionService
     Task<bool> SessionDataExistsAsync(string preKey = DefaultPreKey);
     Task<bool> AnySessionDataExistsAsync();
     Task ClearAllAsync();
+    Task<bool> IsDuplicateFormPostAsync(string formPostTimestampKey = "FormPostTimestamp");
+    Task StartFormPostProcessingAsync(string formPostTimestampKey = "FormPostTimestamp");
+    Task SetFormPostResponseAsync<T>(T postResponseModel, string formPostModelKey = "FormPostModelKey");
+    Task<T> GetPreviousFormPostResponseAsync<T>(string formPostModelKey = "FormPostModelKey");
+    Task SetAsync<T>(string key, T value, string preKey = DefaultPreKey);
```

Comment:
If we're not using it in any context other than DistributedSessionService, it seems unnecessary.

Reply:
I think that these may be used in the future, but agree, if not being used for now they can be removed from interface - so I have done that.